### DISCUSSION TITLE
Print additional to ID, loginname@host at the windows title

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#5820] [General]
Dear Sirs, Dear @rustdesk 
I think that it will be very helpful to have the loginname@host name appeared at the remote windows title.
What I mean is see the current title:
![rustdesk1](https://github.com/rustdesk/rustdesk/assets/15084551/61a71b29-9690-4f20-a507-37e783f6d00f)
Show only the note ID see the details of this note:
![rustdesk2](https://github.com/rustdesk/rustdesk/assets/15084551/33ba323e-0e5b-4483-878e-51a75bca7c47)
What I asking is to have the Windows title more informative by including the info as (in my example):
232047572  kranidis@kramidis
By have it, I can easyly recognise the remote node.
What do you think, Is it possible?

Thanks and Best Regards,
Mike Kranidis

**Comment by rustdesk** [comment]
@21pages what title does anydesk/teamviewer use?

**Reply by 21pages** [reply]
![image](https://github.com/rustdesk/rustdesk/assets/14891774/28b79190-6e9f-4f40-837e-fb1b8f846c90)
![3a002f3abb4737658969ba00375b2a2](https://github.com/rustdesk/rustdesk/assets/14891774/48bbae14-15b1-456f-ad16-5f849391619e)
![40584111bb3831dad4f0b55df54cde5](https://github.com/rustdesk/rustdesk/assets/14891774/71691837-c2ae-4f62-a881-4d3e41715793)

AnyDesk show id or alias, teamviewer show hostname

**Reply by rustdesk** [reply]
> AnyDesk show id or alias

This is we followed

**Reply by rustdesk** [reply]
let's add hostname following id(alias), be care of overflow.

**Reply by rustdesk** [reply]
if `id.lower().contains(hostname.lower())`, show id only.

**Reply by 21pages** [reply]
#5821, ready in nightly build.

**Comment by mikekgr** [comment]
I just tested. It is so lovely thanks !

**Comment by mikekgr** [comment]
Dear Sirs,
today in one of my remotes (Windows 10 both local and remote) I don't get the proper windows title, just note that both ends are on the latest nightly ...
![no-working-title](https://github.com/rustdesk/rustdesk/assets/15084551/a3904e7f-4c50-432a-b430-90b96faa3923)

![no-working-title-remote-screen](https://github.com/rustdesk/rustdesk/assets/15084551/3bbb1870-6db1-42e9-9f05-f7c2e717052c)

Can you guess why? Is it something in the remote (computer) name that is: spare7280-biof ?
Please have a look and let me know if you need something from my side.

Thanks and Best Regards,
Mike Kranidis

**Reply by 21pages** [reply]
Thanks for your feedback, I find one possible reason and will try to fix it.

**Reply by mikekgr** [reply]
Many thanks !

**Comment by mikekgr** [comment]
I can confirm that the today nightly release solve the problem.
Again many thanks 👍
